src/Avalonia.Diagnostics/DevToolsExtensions.cs

```diff
@@ -27,5 +27,15 @@ public static void AttachDevTools(this TopLevel root, KeyGesture gesture)
         {
             DevTools.Attach(root, gesture);
         }
+
+        /// <summary>
+        /// Attaches DevTools to a window, to be opened with the specified options.
+        /// </summary>
+        /// <param name="root">The window to attach DevTools to.</param>
+        /// <param name="options">additional settint of DevTools</param>
```

Comment:
Spelling 'Settint' -> 'Settings' and probably should fix capitalization and period.